_playwright-tests/UI/SnapshotRepo.spec.ts

```diff
@@ -35,15 +35,14 @@ test.describe('Snapshot Repositories', () => {
 
     await test.step('Filter by architecture', async () => {
       await page.getByRole('button', { name: 'filter architecture' }).click();
-      await page.getByRole('option', { name: 'x86_64' }).click();
+      await page.getByTestId('filter_x86_64').click();
```

Comment:
if this is only possible to target with testid than that sucks big time, I wonder is it a configuration issue on our side or if it really coming from patternfly as I am getting this error in the console on clicking any dropdown options/items 🤔 
we should investigate this, if not in this PR, then in a follow up 🔍 

![2025-05-21-150542_hyprshot](https://github.com/user-attachments/assets/1eafc791-3d91-4fc1-b872-f212e315ecd5)

Reply:
opened this for removing the deprecated components: https://issues.redhat.com/browse/HMS-8602